package.json

```diff
@@ -61,6 +61,7 @@
 		"@types/mocha": "^7.0.2",
 		"@types/mustache": "^4.0.1",
 		"@types/node": "^13.11.1",
+		"@types/normalize-path": "^3.0.0",
```

Comment:
Do we need 3PPs for types modules?

Reply:
I think yes. We'll need 3PP for all dependencies for the major versions we use.

Reply:
Here's the approved 3PP.